contracts/src/v0.8/keystone/OCR3Capability.sol

```diff
@@ -70,19 +53,15 @@ contract OCR3Capability is ConfirmedOwner, OCR2Abstract {
     bytes memory _offchainConfig
   ) external override checkConfigValid(_signers.length, _transmitters.length, _f) onlyOwner {
     // Bounded by MAX_NUM_ORACLES in OCR2Abstract.sol
-    for (uint256 i = 0; i < _signers.length; i++) {
+    for (uint256 i = 0; i < _signers.length; ++i) {
       if (_transmitters[i] == address(0)) revert InvalidConfig("transmitter must not be empty");
       // add new signers
       bytes calldata publicKeys = _signers[i];
       uint16 offset = 0;
       uint16 len = uint16(publicKeys.length);
       // scan through public keys to validate encoded format
       while (offset < len) {
-        // solhint-disable-next-line no-unused-vars
-        uint8 keyType = uint8(publicKeys[offset]);
```

Comment:
@DeividasK what was this for? Feels like it's just wasting gas?

Reply:
@archseer @DeividasK does this new format work? No silent reverts but explicit errors